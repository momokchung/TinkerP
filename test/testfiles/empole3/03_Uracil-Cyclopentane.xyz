    27 42_Uracil-Cyclopentane_1.00
     1  N      0.195730   -0.844689    0.823846   178    11     3     2
     2  H      0.450398   -1.796753    1.049768   182     1
     3  C     -1.179049   -0.573684    0.759483   177     5     1     4
     4  O     -1.993646   -1.456265    0.966901   185     3
     5  C     -1.476715    0.811156    0.437560   180     7     6     3
     6  H     -2.506356    1.115651    0.363895   183     5
     7  C     -0.468113    1.682962    0.234891   176     8     9     5
     8  H     -0.638435    2.721643   -0.006164   186     7
     9  N      0.845629    1.305991    0.326831   179    10     7    11
    10  H      1.589693    1.968879    0.185960   181     9
    11  C      1.254261    0.019462    0.636244   175     9    12     1
    12  O      2.422304   -0.301716    0.731879   184    11
    13  C      1.057601   -0.863989    4.427229    10    14    16    15    25
    14  H      1.511454   -0.956045    3.439247    11    13
    15  H      1.602103   -1.527969    5.096427    11    13
    16  C      1.111914    0.601963    4.860165    10    17    13    19    18
    17  H      2.070204    1.074862    4.649441    11    16
    18  H      0.923799    0.683601    5.933388    11    16
    19  C     -0.055437    1.214778    4.089394    10    20    22    16    21
    20  H      0.218867    1.303559    3.035922    11    19
    21  H     -0.339852    2.205919    4.441028    11    19
    22  C     -1.172382    0.177206    4.260422    10    23    19    25    24
    23  H     -1.897921    0.200820    3.449050    11    22
    24  H     -1.716467    0.381903    5.182671    11    22
    25  C     -0.449345   -1.189342    4.364079    10    26    13    22    27
    26  H     -0.692010   -1.833498    3.520720    11    25
    27  H     -0.764451   -1.716744    5.263166    11    25
