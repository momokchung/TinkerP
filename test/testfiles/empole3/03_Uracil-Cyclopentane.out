
     ######################################################################
   ##########################################################################
  ###                                                                      ###
 ###            Tinker  ---  Software Tools for Molecular Design            ###
 ##                                                                          ##
 ##                      Version 8.10.5   February 2023                      ##
 ##                                                                          ##
 ##               Copyright (c)  Jay William Ponder  1990-2023               ##
 ###                           All Rights Reserved                          ###
  ###                                                                      ###
   ##########################################################################
     ######################################################################


 Intermolecular Energy :              -2.53004598 Kcal/mole

 Total Potential Energy :            -51.58584111 Kcal/mole

 Energy Component Breakdown :           Kcal/mole        Interactions

 Atomic Multipoles                   -51.58584111              276
